public/style.css

```
@@ -480,11 +488,32 @@ div, form, h3, h4, select, button {
   text-decoration: inherit;
 }
 
-.no-time-slot-warning {
+.no-time-slot-warning, .successful-booking-icon {
   position: relative;
   height: 24px;
   width: 24px;
   pointer-events: none;
   left: 0px;
   margin: 0px 10px 0px 0px;
-}
\ No newline at end of file
+}
+
+.sucessful-booking {
```

Spelling error... quick fix. Also update the className in other parts of your code

```suggestion
.successful-booking {
```